exporter/sematextexporter/factory.go

```diff
@@ -46,11 +49,17 @@ func createDefaultConfig() component.Config {
 		},
 		QueueSettings: exporterhelper.NewDefaultQueueConfig(),
 		MetricsConfig: MetricsConfig{
+			MetricsEndpoint: usMetricsEndpoint,
 			MetricsSchema:   common.MetricsSchemaTelegrafPrometheusV2.String(),
 			PayloadMaxLines: 1_000,
 			PayloadMaxBytes: 300_000,
 		},
+		LogsConfig: LogsConfig{
+			LogsEndpoint: usLogsEndpoint,
+			AppToken:     logsAppToken,
```

Comment:
Why set a default app token and logs endpoint? They should be empty by default. LogsEndpoint should be set in `Config.Validate` similar to MetricsEndpoint. AppToken should be required to be set by the user, otherwise fail config validation.